app/src/main/java/org/listenbrainz/android/ui/navigation/TopBar.kt

```diff
@@ -1,4 +1,4 @@
-package org.listenbrainz.android.ui.components
+package org.listenbrainz.android.ui.navigation
```

Comment:
Move this back to the components folder. This is a reusable component and not a part of the navigation.

Reply:
Pleaseee just let this be here because it just seems extremely intuitive to have this here. Also we don't use it anywhere else.

Reply:
xD I'm think I'm really really really really fine with this change :)))